man/extensions.doc

```diff
@@ -428,16 +428,10 @@ undefined.  See also read_string/5.
 This predicate is functionally equivalent to sub_atom/5, but operates on
 strings. Note that this implies the string \emph{input} arguments can be
 either strings or atoms.  If \arg{SubString} is unbound (output) it is
-unified with a string. The following example splits a string of the form
-<name>=<value> into the name part (an atom) and the value (a string).
+unified with a string.
+
+For examples of using this predicate, see sub_atom/5.
 
-\begin{code}
```

Comment:
Why delete this example?   Yes, it is now almost a duplicate, but this is one thing strings are meant for.

Reply:
It has been moved to the documentation for sub_atom/5, so that all the examples are together. If you disagree, I can move it back.